### ISSUE TITLE
fontsize does not propagate into PolarAxis

### ISSUE THREAD

**Issue by pzabka** [MakieOrg/Makie.jl#3928]
The parameter fontsize does not propagate into PolarAxis

This works:
```julia
fig = Figure()
ax = PolarAxis(fig[1, 1]; rticklabelsize = 50, thetaticklabelsize = 50)
fig
```

This doesn't:
```julia
fig = Figure(fontsize = 50)
ax = PolarAxis(fig[1, 1])
fig
```

Tested with:
GLMakie v0.10.2 
CairoMakie v0.12.2

**Comment by ffreyer** [comment]
Should be easy to fix by changing the relevant attributes to inherit from fontsize like Axis. See src/makielayout/types.jl

**Comment by ffreyer** [comment]
Fixed by #4314
